### ISSUE TITLE
Can't uninstall themes Ubuntu 20.04.2

### ISSUE THREAD

**Issue by FerdyaVolkov** [vinceliuice/grub2-themes#133]
When I run the script to uninstall the themes it shows me the next message 

Checking for the existence of themes directory...
  
 tela grub theme not exist!

**Comment by vinceliuice** [comment]
Is the theme name right？

**Comment by FerdyaVolkov** [comment]
Yes, I wrote the theme name correctly. I even installed a new one to try to uninstall it. Unfortunately, It didn't work too. I manually deleted the folders inside the grub directory. I don't know if it is enough. I checked the code in the file install.sh but I don't know what exactly is wrong. It must be just a simple thing. I am going to check later. If I find the solution I will let you know. Maybe you can fix it faster than I.

**Comment by hyttmi** [comment]
#180 fixes this.
